sdk/communication/Azure.Communication.Chat/tests/ChatClients/ChatClientsLiveTest.cs

```diff
@@ -37,48 +37,49 @@ public ChatClientsLiveTest(bool isAsync) : base(isAsync)
 
         /// <summary>
         /// Thread      : Create, Get, Update, Delete
-        /// Member      : Add, Update, Remove
+        /// Participant      : Add, Update, Remove
```

Comment:
nit: whitespace indentation (probably worth removing the indentation all together?)